loading/lazyload/move-element-and-scroll.tentative.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<head>
+  <title>Images with loading='lazy' load being moved to another document and then scrolled to.</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="common.js"></script>
+</head>
+
+<!--
+Marked as tentative until https://github.com/whatwg/html/pull/3752 is landed.
+-->
+
+<body>
+  <img id="in_viewport" src='resources/image.png'>
+  <div style="height:10000px;"></div>
+  <img id="below_viewport" src='resources/image.png' loading="lazy">
```

Comment:
I think the src should be a different image, or equivalently the same image with a query param. I think per spec, the first request will create an entry for `image.png` in the [list of available images](https://html.spec.whatwg.org/multipage/images.html#list-of-available-images), and the lazy load request for the same image will match the entry in that cache before it waits to be scrolled in. We should probably have a test for that case too actually..

Reply:
Understood, fixed using query.

Reply:
Thanks!